org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/ADTreeOnDropListener.java

```diff
@@ -213,6 +196,15 @@ private void moveNode(DefaultTreeNode<Object> movingNode, DefaultTreeNode<Object
 		}
 	}
 	
+	private void updateNodePO(MTreeNode mtnParentNode, MTreeNode mtnMovingNode, int NodeIndex, String trxName) {
+		StringBuilder whereClause = new StringBuilder("AD_Tree_ID=").append(mTree.getAD_Tree_ID())
+				.append(" AND Node_ID=").append(mtnMovingNode.getNode_ID());
+		PO mnPO = new Query(Env.getCtx(), mTree.getNodeTableName(), whereClause.toString(), trxName).first();
+		mnPO.set_ValueNoCheck("Parent_ID", mtnParentNode.getNode_ID());
+		mnPO.set_ValueNoCheck("SeqNo", NodeIndex);
+		mnPO.save(trxName);

```

Comment:
Please use saveEx(trxName) instead since save(trxName) can fail silently.

Reply:
Changes have been made.
Thank you for your help and advice.